pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -71,12 +72,28 @@ def __init__(self, monitor: str = 'val_loss', min_delta: float = 0.0, patience:
             self.mode = 'auto'
 
         self.min_delta *= 1 if self.monitor_op == torch.gt else -1
+        self.best = torch_inf if self.monitor_op == torch.lt else -torch_inf
+
+    def state_dict(self):
```

Comment:
```suggestion
    @property
    def state_dict(self):
```

Reply:
wanted to keep the same API as pytorch which exposes this as a method not a property